### ISSUE TITLE
print_dbt_project_evaluator_issues bug

### ISSUE THREAD

**Issue by ZhouSu89** [dbt-labs/dbt-project-evaluator#305]
### Describe the bug

The macro `print_dbt_project_evaluator_issues` does not take the parameters `schema_project_evaluator` and  `db_project_evaluator`. These two parameters are defined but are never used in the macro. Because of this, it is not possible to set schema or database to display violations in the log.

### Steps to reproduce

In dbt-core 1.3.1 with BigQuery adaptor, with dbt-project-evaluator 0.4.0.
Add
```
on-run-end: "{{ dbt_project_evaluator.print_dbt_project_evaluator_issues(schema_project_evaluator=target.dataset, db_project_evaluator=target.project) }}"
```
to `dbt_project.yml` and run 
```
dbt build --select package:dbt_project_evaluator
```
in the terminal.
### Expected results

Logs on rule violations printed in the terminal after the build command finishes.
### Actual results

A database error saying `Table "fct_documentation_coverage" must be qualified with a dataset`
### Screenshots and log output

![Screenshot 2023-02-10 at 10 15 47](https://user-images.githubusercontent.com/35170396/218052899-98914458-15e0-4fe7-a5f7-b2655cc5b663.jpg)

### System information
**The contents of your `packages.yml` file:**
```
packages:
  - package: dbt-labs/dbt_utils
    version: 1.0.0
  - package: calogica/dbt_expectations
    version: [">=0.8.0", "<0.9.0"]
  - package: dbt-labs/metrics
    version: 1.3.2
  - package: dbt-labs/audit_helper
    version: 0.6.0
  - package: dbt-labs/dbt_project_evaluator
    version: 0.4.0
```
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ x] bigquery
- [ ] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
Core:
  - installed: 1.3.1
  - latest:    1.4.1 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - bigquery: 1.3.0 - Update available!

```

### Are you interested in contributing the fix?

Yes

**Comment by dave-connors-3** [comment]
hey @ZhouSu89 ! Thanks so much for opening this issue - we actually have an open (soon-to-be-merged!) PR #300 that will clean up the handling of db and schema variables for the models built by this package in the optional on-run-end hook. If you'd like to give that logic a try to see if that addresses your issue, that would be awesome!

[Check out the docs on installing git packages if you want to give it a spin!](https://docs.getdbt.com/docs/build/packages#git-packages)

**Comment by katieclaiborne** [comment]
Thank you both! We've confirmed that the logic in #300 resolves the issue in our project (dbt-bigquery==1.3.0).

cc @nathanielburren and @emilee-jurkowski

**Comment by pei0804** [comment]
This is a great solution.

**Comment by b-per** [comment]
Fixed with #300
